main.js

```diff
@@ -0,0 +1,66 @@
+var PlayGame = {
+  
+  board:  [[5, 4, 3, 2, 1], [], []],
+  results: function () {
+    for (let i=0; i < this.board.length; i++) {
+      const rowResults = this.board[i].map(num => num.toString()).join(" ");
+      console.log ("---" + " "  + rowResults);
+    };
+  },
+  
+  checkWinner: function () {
+    const winningPeg = [5, 4, 3, 2, 1];
+    for (let i = 1; i < this.board.length; i++) {
+      const checkDiscsRow = this.board[i];
```

Comment:
it is bad practice to declare variables inside of a for loop because they get created/destroyed in the loop, so you won't be able to use it outside the loop